CHANGELOG.rst

```diff
@@ -6,7 +6,8 @@
 * Text documents without any doctests no longer appear as "skipped".
   Thanks `@graingert`_ for reporting and providing a full PR (`#1580`_).
 
-*
+* Fix internal error issue when `method` argument is missing for 
+  `teardown_method`. Fixes (`#1605`).
```

Comment:
Some small ReST issues with this:

- Use double backticks for monospace
- Probably want to drop the parens
- You'll need to add `#1605` to the link list further down